### PR TITLE
Updating multiple feeds page for developer guide's content card section

### PR DESCRIPTION
**Pull request by douglas-putnam**
# Pull Request/Issue Resolution

**Description of Change:**
> I added swift code snippets to the /developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds/ page

**Reason for Change:**
> The page was missing code snippets

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Feature Release Date:__)
- [x] No

> If yes, please note the date of the feature release.

---
---

## PR Checklist
- [x] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [x] Tag @EmilyNecciai as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [x] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide).
- [x] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices).
- [x] Tag others as Reviewers as necessary.
- [x] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

## ATTENTION: REVIEWERS OF THIS PR
- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
---
---

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                                               name:ABKContentCardsProcessedNotification
+                                             object:nil];
+```
+
+{% endtab %}
+{% tab SWIFT %}
+
+```swift
+NotificationCenter.default.addObserver(self, selector:
+  #selector(contentCardsUpdated),
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: should this be on the previous line?

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+{% endtab %}
+{% tab SWIFT %}
+
+```swift
+@objc private func contentCardsUpdatedNotifcationReceived(notification: NSNotification) {
+    guard let updateSuccessful = notification.userInfo?[ABKContentCardsProcessedIsSuccessfulKey] as? Bool else { return }
+    if updateSuccessful {
+        // Get an array containing only cards that have the "Transactional" feed type set in their extras.
+        let filteredArray = getCards(forFeedType: "Transactional")
+        NSLog("Got filtered array of length: %@",filteredArray?.count ?? 0)
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: Super nit: space before `filteredArray`

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+        // Pass filteredArray to your UI layer for display.
+    }
+}
+
+func getCards(forFeedType type: String) -> [ABKContentCard]? {
+    guard let allCards = Appboy.sharedInstance()?.contentCardsController.contentCards as? [ABKContentCard] else { return nil }
+    // return filtered cards
+    return allCards.filter {
+        if $0.extras?["feed_type"] as? String == type {
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: Swift looks pretty cool 😎

Comment (douglas-putnam) [reply _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: :)

File: _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md (hunk 1 above)
Comment (douglas-putnam) [reply _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: Just followed same pattern as https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/content_cards/data_model/

Comment (douglas-putnam) [reply _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: FWIW, this is how editor formats it. If you prefer bring it up, let me know and I will make the change to both docs.

Comment (hokstuff) [reply _docs/_developer_guide/platform_integration_guides/ios/content_cards/multiple_feeds.md]: I also noticed it was in the same format [in News Feed as well](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/news_feed/news_feed_data_model/#getting-the-data)

We can just leave this format as-is IMO. We can update both the NF and CC formatting separately

**Comment by EmilyNecciai** [comment]
@cla-bot check

**Pull request merged**